http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -106,13 +104,22 @@ case class AuditInfo(creator: WorkbenchEmail,
                      dateAccessed: Instant,
                      kernelFoundBusyDate: Option[Instant])
 
-sealed trait ClusterTool extends EnumEntry with Serializable with Product
+sealed trait ClusterTool extends EnumEntry with Serializable with Product {
```

Comment:
This probably conflicts with https://github.com/DataBiosphere/leonardo/pull/1124.. will cross that bridge when I come to it.